.github/workflows/backport.yml

```
@@ -5,33 +5,38 @@ on:
    - main
    - v/*
 jobs:
-  get-maintenance-branches:
+  get-maintenance-versions:
     runs-on: ubuntu-latest
     outputs:
-      branches: "['5.3', '5.4', '5.5']"
+      versions: ${{ steps.get-maintenance-versions-and-branches.outputs.versions-and-branches }}
```

Maybe use `versions-and-branches` instead `versions` here and in the other places?
```suggestion
      versions-and-branches: ${{ steps.get-maintenance-versions-and-branches.outputs.versions-and-branches }}
```

---

https://github.com/hazelcast/hz-docs/pull/1651/commits/4b929f53d24767edfb1d871f6d56c06bcfd4ddfd